VideoLocker/src/main/java/org/edx/mobile/task/Task.java

```diff
@@ -49,6 +45,7 @@ public Task(Context context) {
 
     public void setProgressDialog(ProgressBar progressBar) {
         this.progressBar = progressBar;
+        this.progressCallback = null;
```

Comment:
Since we have already given the Task a custom `ProgressBar` by calling this function, we need to take care of the default the progress indicator, so that 2 progress indicators never show up.